.github/workflows/commit-message.yml

```
@@ -0,0 +1,49 @@
+name: '📝 Commit Message Lint'
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)